packages/api/src/controllers/RestRepository.ts

```diff
@@ -22,7 +22,7 @@ export abstract class RestRepository<T> {
   }
   registerRoutes(handlers: IRouteMap): IRouteMap {
     const fallbackHandlers = {
-      [this.listRoute]: this.getAll.bind(this)
```

Comment:
This `bind` was a leftover from a previous attempt at awilix injection of class methods.